sdk/python/packages/flet-core/src/flet_core/matplotlib_chart.py

```diff
@@ -133,7 +135,12 @@ def _before_build_command(self):
         super()._before_build_command()
         if self.__figure is not None:
             s = io.StringIO()
-            self.__figure.savefig(s, format="svg")
+
+            if self.__transparent:
+                self.__figure.savefig(s, format="svg", transparent=True)
+            else:
+                self.__figure.savefig(s, format="svg")
+
```

Comment:
Why not just `self.__figure.savefig(s, format="svg", transparent=self.__transparent)`?

Reply:
I would agree that's better